docs/src/use/formatters/index.md

```diff
@@ -4,7 +4,7 @@ eleventyNavigation:
     key: formatters
     parent: use eslint
     title: Formatters Reference
-    order: 6
+    order: 7
```

Comment:
We should update this in the template too:

https://github.com/eslint/eslint/blob/53b1ff047948e36682fade502c949f4e371e53cd/templates/formatter-examples.md.ejs#L7